src/store/getters/state-getters.ts

```diff
@@ -234,59 +241,82 @@ export const isPeopleAndRoleValid = (state: any): boolean => {
 //
 
 /** Whether File and Pay button should be enabled. */
-export const isEnableFilePayBtn = (state: any, getters: any): boolean => {
-  const step1Valid = state.stateModel.defineCompanyStep.valid
-  const step2Valid = state.stateModel.peopleAndRoles.valid
-  const step3Valid = getters.isTypeBcomp ? state.stateModel.createShareStructureStep.valid : false
+export const isEnableFilePayBtn = (state: StateIF, getters: any): boolean => {
+  const step1Valid = isDefineCompanyStepValid(state)
+  const step2Valid = isPeopleAndRoleValid(state)
+  const step3Valid = getters.isTypeBcomp ? state.stateModel.shareStructureStep.valid : false
   const step4Valid = getters.isTypeBcomp ? state.stateModel.incorporationAgreementStep.valid : false
   const step5Valid = state.stateModel.certifyState.valid && state.stateModel.incorporationDateTime.valid
   return (step1Valid && step2Valid && step3Valid && step4Valid && step5Valid)
```

Comment:
I'm probably to convert this to separate getters and move the final logic to the Actions component. Thoughts?